docs/streams/tutorial.html

```diff
@@ -62,7 +62,7 @@
     </pre>
 
     <p>
-        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-SNAPSHOT</code> to <code>0.11.0.1</code>.
+        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-cp1</code> to <code>0.11.0.1</code>.
```

Comment:
Collateral damage, don't think it matters though.

Reply:
Should this stay at -SNAPSHOT? In the actual release, there wouldn't be snapshot, and so perhaps this line should be removed instead? If removed, that might get merged into dev branches accidentally though.

Reply:
I usually update just gradle.properties and the test. Looks like you've got it.

Docs changes won't matter since they're not served out of this repo or branch, but you're right that someone could accidentally pick up the change.